server/app-engine/gtfsrt.py

```diff
@@ -176,12 +176,30 @@ def make_trip_update(list):
 
     return entity
 
-def alert_is_current(alert):
+def alert_is_current(alert, service_alert_ui):
     print('alert_is_current()')
     if not('time_start' in alert and 'time_stop' in alert):
         return False
     now = int(time.time())
-    return now >= int(alert['time_start']) and now < int(alert['time_stop'])
+
```

Comment:
Shouldn't all of these time values already be ints? Why do they keep getting cast to int?

Also, again, unless I'm missing something, this whole portion could be simplified quite a bit. If alerts come in with `time_stop` set to 0, set it to some very high value. Then, instead of the cascaded `if`s:

```
return (now >= int(alert['time_start']) or include_future_alerts) and now < int(alert['time_stop'])
```

Reply:
You're right, I think it's less pretty but it does save a bunch of lines